client/.storybook/daylightTheme.js

```diff
@@ -0,0 +1,20 @@
+import { create } from '@storybook/theming/create';
+
+export default create({
+  base: 'light',
+
+  textPrimaryColor: '#4D4C4C',
+  textSecondaryColor: '#707070',
+  textThirdColor: '#FFFFFF',
+  backgroundPrimaryColor: '#00A8CC',
+  buttonPrimaryColor: '#45BFD9',
+  warningColor: '#FEA630',
+  lightColor: '#FFBE68',
+  inputColor: '#EFEAEA',
+
+  colorPrimary: 'hotpink',
+  colorSecondary: 'deepskyblue',
```

Comment:
you other names are a bit more specific than these, is that intentional?

Reply:
No, these both I should delete.